Generals/CMakeLists.txt

```diff
@@ -8,6 +8,14 @@ add_feature_info(GeneralsTools GENZH_BUILD_GENERALS_TOOLS "Build Generals Mod To
 option(GENZH_BUILD_GENERALS_EXTRAS "Build extra tools/tests for Generals" OFF)
 add_feature_info(GeneralsExtras GENZH_BUILD_GENERALS_EXTRAS "Build Generals Extra Tools/Tests")
 
+# Do we want to build documentation?
+option(GENZH_BUILD_GENERALS_DOCS "Build documentation for Generals" OFF)
```

Comment:
This option does not show in cmake. `GENZH_BUILD_ZEROHOUR_DOCS` does. Perhaps because of the other error.